lib/SearchForm/SearchForm.js

```diff
@@ -25,6 +25,8 @@ export function SearchForm({
   handleChange,
   handleClear,
   handleSubmit,
+  fieldName,
+  idKey,
```

Comment:
Please, follow the [codstyle](https://github.com/folio-org/ui-data-export/blob/master/CODESTYLEGUIDE.md), callbacks should be placed at the end of the destructuring list.

Reply:
done